.changeset/lucky-carrots-stare.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-explore': patch
```

Comment:
We'll need to split this into two separate changesets. The text here doesn't make sense for the `core-components` changes right now